packages/integrations/stores/xml-file/src/configs-db.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
is that file used for testing?
if so, it's better to create a test file, and put it inside a "stub" directory. (you can see a testing structure example in (package/common)